x/pairing/keeper/staking.go

```diff
@@ -72,6 +72,15 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, validator, creator, chainID strin
 				utils.Attribute{Key: "provider", Value: senderAddr.String()},
 			)
 		}
+
```

Comment:
sorry for the hustle, talked with omer and we dont want this change.
can you revert?